src/arrayMethodJoin.js

```diff
@@ -4,8 +4,18 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let string = '';
+
+    for (let i = 0; i < this.length; i++) {
+      let element = this[i];
+
+      element = (element === null || element === undefined) ? '' : `${element}`;
```

Comment:
using ternary, and then giving your var `''` – bad approach, you'd rather use _if_

Reply:
Changed a bit, but still using a ternary, because 1 of 2 need to be returned (direct purpose of ternary to return 1 of 2)